.github/scripts/generate_release_notes.py

```diff
@@ -91,50 +99,70 @@ def get_repo_priority(name):
     githubEnv.write("REL_BRANCH=release-{}\n".format(
         re.search(majorReleaseRegex, releaseVersion).group(1)))
 
+# get dashboard release version
 releases = sorted([r for r in g.get_repo("dapr/dashboard").get_releases()], key=lambda r: r.created_at, reverse=True)
 dashboardReleaseVersion = re.search(dashboardReleaseVersionRegex, releases[0].tag_name).group(1)
 print("Detected Dapr Dashboard version {}".format(dashboardReleaseVersion))
 
-
+# get all cards in all columns
 columns = project.get_columns()
 cards = []
 for column in columns:
     cards = cards + [c for c in column.get_cards()]
 
+# generate changes
 for c in cards:
     issueOrPR = c.get_content()
+    contributors = []
+    url = ""
+    try:
+        pr = issueOrPR.as_pull_request()
```

Comment:
only a PR can be converted to a PR object, otherwise will throw error.

Reply:
Put this comment in the code for future readers?